### ISSUE TITLE
Importing presets with local mods crashes the launcher

### ISSUE THREAD

**Issue by Weilbyte** [muttleyxd/arma3-unix-launcher#97]
When importing a preset which contains local (instead of Steam) mods, the launcher will crash.

```
arma3-unix-launcher: /usr/include/nlohmann/json.hpp:17944: const value_type& nlohmann::basic_json<ObjectType, ArrayType, StringType, BooleanType, NumberIntegerType, NumberUnsignedType, NumberFloatType, AllocatorType, JSONSerializer>::operator[](T*) const [with T = const char; ObjectType = std::map; ArrayType = std::vector; StringType = std::__cxx11::basic_string<char>; BooleanType = bool; NumberIntegerType = long int; NumberUnsignedType = long unsigned int; NumberFloatType = double; AllocatorType = std::allocator; JSONSerializer = nlohmann::adl_serializer; nlohmann::basic_json<ObjectType, ArrayType, StringType, BooleanType, NumberIntegerType, NumberUnsignedType, NumberFloatType, AllocatorType, JSONSerializer>::const_reference = const nlohmann::basic_json<>&; nlohmann::basic_json<ObjectType, ArrayType, StringType, BooleanType, NumberIntegerType, NumberUnsignedType, NumberFloatType, AllocatorType, JSONSerializer>::value_type = nlohmann::basic_json<>]: Assertion `m_value.object->find(key) != m_value.object->end()' failed.
zsh: abort (core dumped)  ./src/arma3-unix-launcher/arma3-unix-launcher
```

**Comment by muttleyxd** [comment]
Can you post an example HTML preset for this? I didn't get to use much of original launcher so HTML import only supports steam mods

**Comment by Weilbyte** [comment]
Here's an example one. 
[arma3localpreset.html.txt](https://github.com/muttleyxd/arma3-unix-launcher/files/4584374/arma3localpreset.html.txt)

**Comment by muttleyxd** [comment]
Thanks!

```
<span class="whups" data-type="Link" data-meta="local:ACE Compat - RHS United States Armed Forces|@ACE Compat - RHS United States Armed Forces|" />
```

Is `@ACE Compat - RHS United States Armed Forces` a directory inside Arma's game directory?

Should be a pretty simple fix, though I don't know if I'll find some time before weekend

**Comment by Weilbyte** [comment]
Not sure how the BI Launcher handles those. Might be going by the mods name and not directory name?

**Comment by Weilbyte** [comment]
I'm thinking it could go through the mods and parse their mod.cpp until it finds a mod that has the same `description` and then load the mod.

**Comment by muttleyxd** [comment]
Fixed by 59384af.

Feel free to reopen if issue persists
